app/assets/javascripts/components/GlobalNav.js

```diff
@@ -258,6 +267,18 @@ define(['jquery', 'DoughBaseComponent', 'mediaQueries', 'utilities', 'common'],
       })
       .mousedown(function(e) {
         e.preventDefault();
+      }).on('touchstart', function(e) {
+        // remove hover event handler if touch event detected
+        $(e.target).parents('[data-dough-nav-clump-heading]').off('mouseenter')
+      }).on('touchend', function(e) {
+        if (!mediaQueries.atSmallViewport()) {
+          // touch event on clump heading triggers open/close subnav
+          if ($(e.target).parents('[data-dough-nav-clump]').hasClass('is-active')) {
+            self._closeDesktopSubNav($(e.target).parents('[data-dough-nav-clump-heading]'));
+          } else {
+            self._openDesktopSubNav($(e.target).parents('[data-dough-nav-clump-heading]'));
```

Comment:
Can we make `$(e.target).parents('[data-dough-nav-clump-heading]')` a variable to use on the two if statements? Wdyt?